lib/zendesk/ticket.ex

```diff
@@ -65,6 +68,25 @@ defmodule Zendesk.Ticket do
     %Operation{path: "tickets.json", parser: &parse_list/1} |> Operation.with_page_size(1)
   end
 
+  @doc """
+  Returns the full ticket object as it would be after applying the macro to the ticket.
+
+  It doesn't actually change the ticket.
+  """
+  @spec after_changes(non_neg_integer(), non_neg_integer()) :: Operation.t()
```

Comment:
```suggestion
  @spec after_changes(pos_integer() | Ticket.t(), pos_integer() | Macro.t()) :: Operation.t()
```

Reply:
I updated to accept `Ticket.t()` and `Macro.t()` in 594d345.